src/scripts/features/searchbar.ts

```diff
@@ -415,3 +417,12 @@ function focusSearchbar() {
 		domsearchbar?.focus()
 	}
 }
+const searchBar = document.getElementById("searchbar") as HTMLInputElement;
+
+// shortcut to active searchbar
+document.addEventListener("keydown", (event) => {
+  if (event.keyCode === 191) {
```

Comment:
```suggestion
  if (event.key === '/') {
```

`keyCode` is deprecated, best to use `key`